installJDOnSLURM.py

```diff
@@ -0,0 +1,274 @@
+    conn.run(cmd, hide=False)
+    return None
+
+def check_daskconfig(conn):
+    """
+    Check if dask has been configured on remote host
+
+    :param conn: ssh connection object
+    :param args: ArgumentParser return object containing command line arguments. included for optional specification of local port
+    :param platform: platform name
+    :return outfilename: name of slurm output file on remote host
+    """
+    cmd = "if test -f ~/.config/dask/config.yml ; then echo 'True'; fi"
+    daskconfig_exists = False
+    result = conn.run(cmd, hide=True)
+    daskconfig_exists = result.stdout
+    return daskconfig_exists
+
+def daskconfig(conn, platform):
+    """
+    Submit batch job with jupyter server on remote host
+
+    :param conn: ssh connection object
+    :param args: ArgumentParser return object containing command line arguments. included for optional specification of local port
+    :param platform: platform name
+    :return outfilename: name of slurm output file on remote host
+    """
+    cmd = f"cd {remoteJDD} && mkdir -p ~/.config/dask && cp -r config/dask/config_{platform}.yml ~/.config/dask/config.yml"
+    conn.run(cmd, hide=True)
+    return None
+
+
+def install_JD(config_inputs, platform_name, envfile):
+    #Clone folder as needed
+    folder_exists = ssh_remote_executor(config_inputs, check_clone)
+    if not folder_exists:
+        print ('Cloning JupyterDaskonSLURM on remote host...')
+        ssh_remote_executor(config_inputs, clone_folder)
+        folder_exists = ssh_remote_executor(config_inputs, check_clone)
+        if not folder_exists:
+            raise ValueError(f'Error cloning repository. Check git credentials or clone manually')
+        
+    #Install mamba as needed
+    mamba_exists = ssh_remote_executor(config_inputs, test_mamba)
+    if not mamba_exists:
+        print ('Installing mamba on remote host...')
+        ssh_remote_executor(config_inputs, install_mamba)
+        mamba_exists = ssh_remote_executor(config_inputs, test_mamba)
+        if not mamba_exists:
+            raise ValueError(f'Error installing mamba. Please install manually')
+        
+    #Create environment
+    env_exists, envname = ssh_remote_executor(config_inputs, test_env, envfile)
+    if not env_exists:
+        print (f'Creating mamba environment from {envfile} on remote host...')
+        ssh_remote_executor(config_inputs, create_env, envfile)
+        env_exists = ssh_remote_executor(config_inputs, test_env, envfile)
+        if not env_exists:
+            raise ValueError(f'Error creating environment. Please create manually')
+        
+    #Configure Jupyter
+    jpconfig_exists = ssh_remote_executor(config_inputs, check_jpconfig)
+    if not jpconfig_exists:
+        print ('Configuring Jupyter on remote host...')
+        ssh_remote_executor(config_inputs, jpconfig, envname)
```

Comment:
Had some issues in filling in the password - it asked for this repeatedly, and finally took in only an empty string. Anybody else had the same issue?

Reply:
Couldn't replicate the issue running from mac. Pinging @cpranav93  just fyi